### ISSUE TITLE
Not all videos are being saved...

### ISSUE THREAD

**Issue by dtitov** [aunum/gold#14]
Hi, 

I'm running a lot of episodes of LunarLander in a loop (maybe hundreds). When I call `environment.End()` in the end, I get all the episodes information printed, like this:
```
...

  "15":
    episode_id: 15
    episode_length: 80
    reward: 20.416142
    timestamp:
      seconds: 1588666897

...
```

But, for some reason, it records only a few videos:
```
Videos:
  "0":
    content_type: video/mp4
  "1":
    content_type: video/mp4
    episode_id: 1
  "8":
    content_type: video/mp4
    episode_id: 8
  "27":
    content_type: video/mp4
    episode_id: 27
```

I'm having a hard time trying to find the parameter that can turn on saving video for each and every episode. Is it even possible?

**Comment by pbarker** [comment]
Yep so thats the sampling rate https://github.com/aunum/sphere/blob/master/api/v1alpha/env.proto#L290 it defaults to cubic, I haven't yet parameterized that here https://github.com/aunum/gold/blob/master/pkg/v1/env/env.go#L89 but would be a small fix.

I'm pretty busy with work right now so it may take a second to get to that, but feel free to throw up an MR if you feel up for it.

**Comment by dtitov** [comment]
I’m not very familiar with Protobuf, so I’ll just wait for you :) Thanks for the info!
